src/test/java/uk/gov/register/resources/RedirectResourceTest.java

```diff
@@ -40,23 +41,44 @@ public void getItemRedirect() throws Exception {
     }
 
     @Test
-    public void getV1Redirect() throws Exception {
-        Response response = register.getRequest(TestRegister.register, "/v1", WILDCARD);
-        assertThat(response.getStatus(), equalTo(307));
+    public void getV1RedirectAcceptHtml() throws Exception {
+        Response response = register.getRequest(TestRegister.register, "/v1", MediaType.TEXT_HTML);
+        assertThat(response.getStatus(), equalTo(301));
         assertThat(response.getLocation().getPath(), equalTo("/"));
     }
 
+    @Test
+    public void getV1RedirectJsonExtension() throws Exception {
+        Response response = register.getRequest(TestRegister.register, "/v1.json", WILDCARD);
+        assertThat(response.getStatus(), equalTo(301));
+        assertThat(response.getLocation().getPath(), equalTo("/register.json"));
+    }
+
+    @Test
+    public void getV1RedirectAcceptJson() throws Exception {
+        Response response = register.getRequest(TestRegister.register, "/v1", MediaType.APPLICATION_JSON);
+        assertThat(response.getStatus(), equalTo(301));
+        assertThat(response.getLocation().getPath(), equalTo("/register"));
+    }
+
+    @Test
+    public void getNextRedirectJsonExtension() throws Exception {
+        Response response = register.getRequest(TestRegister.register, "/next.json", WILDCARD);
+        assertThat(response.getStatus(), equalTo(301));
+        assertThat(response.getLocation().getPath(), equalTo("/next/register.json"));
+    }
+
     @Test
     public void getV1ItemRedirect() throws Exception {
         Response response = register.getRequest(TestRegister.register, "/v1/items/sha-256:9432331d3343a7ceaaee46308069d01836460294c672223b236727a790acf786", WILDCARD);
-        assertThat(response.getStatus(), equalTo(307));
+        assertThat(response.getStatus(), equalTo(301));
```

Comment:
I think this change is not intended by the ADR, i.e. only the root paths should `301` other existing paths should continue to `307`. CC @arnau

Reply:
agree

Reply:
Have changed this back